Sprint-1/implement/max.js

```diff
@@ -1,4 +1,18 @@
 function findMax(elements) {
+  // Filter out non-numeric values
+  const numericElements = elements.filter(
+    (element) => typeof element === "number"
+  );
+
+  // If no numeric elements exist, return -Infinity
+  if (numericElements.length === 0) {
+    return -Infinity;
+  }
+
+  // Return the largest numeric element
+  return Math.max(...numericElements);
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```

Reply:
findMax([-Infinity])// Return value: -Infinity.
findMax([-Infinity, 0])// Return value: 0.
findMax([NaN])// Return value: -Infinity.
findMax([0, NaN, 1])// Return value: 1.

Reply:
Have you tested your function on them to see if they met your expectation. 
Those values were what I expected initially too, but I got some surprised.

Reply:
Apologies for the mistake; I have run the test. 
findMax([0, NaN, 1]) // Return value: NaN